src/main/java/org/broadinstitute/hellbender/tools/spark/utils/IntHistogram.java

```diff
@@ -198,6 +202,46 @@ public boolean isDifferentByKSStatistic( final IntHistogram sampleHistogram, fin
             return false;
         }
 
+        public KSData evalKS( final IntHistogram sampleHistogram ) {
```

Comment:
If this function is to be moved to `FindSmallIndelRegions`, then the associated class `KSData` should be too.